docs/user-manual/build-system/01-cmake-intro.md

```diff
@@ -33,22 +26,14 @@ steps can be found in [API](./cmake-api.md). This document will explain the usag
 
 The CMakeList.txt file defines the steps needed to build **something** in CMake.  In fprime, we use this file to define the source, autocoder, and module dependencies for modules in fprime. A `register_` function is called to tie into the fprime autocoder environment. This keeps fprime modules simple, although all of CMake's power can be used when needed.
 
-Users need only set the `SOURCE_FILES` variable to a list of autocoder and code sources and then call
-`register_fprime_module` to setup a module for fprime (Port/Component). Deployments are done similarly, but since these
-`CMakeLists.txt` files are the entry point to a build, more setup is needed. Deployments should also call
-`register_fprime_executable` as a binary output is desired. `add_fprime_subdirectory` is also used in deployments to
-link to all the other modules they use.
+Users need only set the `AUTOCODER_INPUTS` and `SOURCES` directives to a list of autocoder and code sources as part of the
+`register_fprime_module` call to setup a module for fprime (Port/Component). Deployments are done similarly but use the
```

Comment:
`register_fprime_module` vs `register_fprime_library` - is there a preferred one? If `register_fprime_library` is not preferred should we scrub it?